generate.ts

```diff
@@ -34,7 +34,7 @@ var sdkGeneratorGlobals: SdkGlobals = {
 };
 
 const defaultApiSpecFilePath = "../API_Specs"; // Relative path to Generate.js
-const defaultApiSpecGitHubUrl = "https://raw.githubusercontent.com/PlayFab/API_Specs/master";
+const defaultApiSpecGitHubUrl = "https://raw.githubusercontent.com/PlayFab/API_Specs/TOC";
```

Comment:
>TOC [](start = 85, length = 3)

Just curious, why are we switching to another branch instead of using master?

Reply:
That was a mistake. I was testing.
I thought I reverted it.
Thanks for catching it.